Plugin/Model/SavePlugin.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mooore\WordpressIntegrationCms\Plugin\Model;
+
+use Magento\Cms\Model\PageRepository;
+use Magento\Framework\Controller\ResultInterface;
+use Mooore\WordpressIntegrationCms\Model\HttpClient\Page as PageClient;
+use Mooore\WordpressIntegrationCms\Model\RemotePageRepository;
+use Magento\Cms\Model\Page;
+use Magento\Cms\Helper\Page as PageHelper;
+use Magento\Cms\Api\Data\PageInterface;
+
+class SavePlugin
+{
+    /**
+     * @var PageClient
+     */
+    private $pageClient;
+
+    /**
+     * @var RemotePageRepository
+     */
+    private $remotePageRepository;
+
+    /**
+     * @var PageHelper
+     */
+    private $pageHelper;
+
+    public function __construct(
+        PageClient $pageClient,
+        RemotePageRepository $remotePageRepository,
+        PageHelper $pageHelper
+    ) {
+        $this->pageClient = $pageClient;
+        $this->remotePageRepository = $remotePageRepository;
+        $this->pageHelper = $pageHelper;
+    }
+
+    public function afterSave(PageRepository $subject, Page $result)
```

Comment:
Saving the metadata should only happen when the module has been configured to do so! Also need to check if the API credentials are not empty.

Reply:
Same answer as in the DeletePlugin: The checking of the API credentials happens further down the line in the RemotePageRepository. An error will occur and will be written to the log files for further investigation. Could you explain to me what configuration you would expect to see here?